webpack.config.js

```diff
@@ -49,7 +49,9 @@ const baseConfig = {
         options: {
           babelrc: false,
           presets: [
-            '@babel/preset-typescript',
+            ['@babel/preset-typescript', {
+              allowNamespaces: true
```

Comment:
By default, babel doesn't quite enable typescript namespaces, and it will not support shared scopes in namespaces (see https://babeljs.io/docs/en/next/babel-plugin-transform-typescript.html#caveats)